src/builtin-templates/hosted-payment-card-form.js

```diff
@@ -0,0 +1,32 @@
+import { html } from '/web_modules/lit-html/lit-html.js';
+
+const BASE_URL = 'https://vault.automationcloud.net/forms/index.html';
+const DEFAULT_OPTIONS = {
+    fields: 'pan,name,expiry-select,cvv',
+    brands: 'visa,mastercard,amex',
+    validateOnInput: 'on',
+    css: 'https://ubio-application-bundle-dummy-server.glitch.me/style.css'
+};
+
+export default (otp, formOptions = {}, formStyles = {}) => {
+    const options = {
+        ...DEFAULT_OPTIONS,
+        ...formOptions
+    };
+
+    const optionsString = Object.keys(options)
+        .filter(key => options[key])
+        .map(key => `&${key}=${options[key]}`)
+        .join('');
+
+    const src = `${BASE_URL}?otp=${otp}${optionsString}`;
+
+    return html`
```

Comment:
Let's stick with vanilla DOM stuff for builtin templates now. In theory, it should work just fine to drop the `html` tag here and replace it with a `document.createElement('iframe')`...

Reply:
ok!